cmd/fts/cmd/task.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"sync"
+	"time"
+
+	client "github.com/textileio/powergate/api/client"
+	userPb "github.com/textileio/powergate/api/gen/powergate/user/v1"
+)
+
+// Task is an individual data collection to be stored on Filecoin
+type Task struct {
```

Comment:
becomes the pipeline task object and the log file output format // resume job input format.